src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -114,4 +122,193 @@ class IntervalExpressionSegment(BaseSegment):
             optional=True
         )
     ),
+    # This is a hook point to allow subclassing for other dialects.
+    # In the ANSI dialect this is designed to be a basic starting point.
+    FunctionDefinitionGrammar=Sequence(
+        AnyNumberOf(
+            Sequence(
+                'LANGUAGE',
+                # Not really a parameter, but best fit for now.
+                Ref('ParameterNameSegment'),
+                Sequence(
+                    'OPTIONS',
+                    Bracketed(
+                        Delimited(
+                            Sequence(
+                                Ref('ParameterNameSegment'),
+                                Ref('EqualsSegment'),
+                                Anything()
+                            ),
+                            delimiter=Ref('CommaSegment')
+                        )
+                    ),
+                    optional=True
+                )
+            ),
+            'WINDOW',
```

Comment:
What is this stuff on lines 148-159? I don't recognize it as anything in BigQuery.

Reply:
They're copies of some of the syntax for postgres. I had just assumed that they were still options in Bigquery - my bad. I'll remove them.